docs/UserGuide.md

```diff
@@ -365,11 +366,92 @@ To view each plan in detail, enter
 1. Plan A
 2. Plan B
 3. Grow My Muscles
-
 ----------------------------------------------------------------------
 ```
 In the current plans list above, there are 3 plans available.
 
+---
+### List details of a Plan: `plan /details`
+Lists all the workouts in a plan.
+
+Format: `plan /details <plan number to view details>`
+
+**Example**<br/>
+Now, the plan list contains the following plans:
+```
+----------------------------------------------------------------------
+Here are all your plan(s).
+To view each plan in detail, enter
+'plan /details <plan number in list>'.
+
+1. plan A
+2. plan B
+3. Grow My Muscles
+----------------------------------------------------------------------
+```
+To view details of plan number 3 from the list, enter the following command:
+*Plan number 3 is created by the command in [Create A Plan](#create-a-plan-plan-new).*
+```
+> plan /details 3
+```
+**Expected Outcome**
+```
+----------------------------------------------------------------------
+Here are the 4 workouts in [Grow My Muscles].
+1. push up (10 reps)
+2. sit up (10 reps)
+3. pull up (10 reps)
+4. push up (10 reps)
+----------------------------------------------------------------------
+```
+---
```

Comment:
Perhaps could add a short explanation on the expected outcome. E.g. "All workouts in the Grow My Muscles plan will be displayed....."